### DISCUSSION TITLE
Are there EAF applications for replacing Microsoft Office word, excel ...  / Libre Office Writer, Calc ...?

### DISCUSSION THREAD

**Discussion by iagogv3** [emacs-eaf/emacs-application-framework#1125] [Q&A]
Hi,

I am new on this forum, so sorry for my ignorance. I believe (Microsoft/Libre) Office documents created with word processors can be viewed in Emacs through DocView mode and there are modes also for spreadsheet files. Here I would like to ask if these kind of files are editable in the context of EAF.

Thanks!

**Comment by manateelazycat** [comment]
Although PyQt can develop Office software, such as WPS Office in China, which was developed by Qt, I think that an Office software requires millions of lines of code, which is not something that can be developed by the community
